config/opendatahub-io/org.yaml

```diff
@@ -179,210 +180,27 @@ orgs:
         privacy: closed
         repos:
           s2i-lab-elyra: maintain
-      repo-kserve-write:
-        description: "Members with Write access to the kserve repository"
-        members:
-        - DaoDaoNoCode
-        - DharmitD
-        - HumairAK
-        - Jooho
-        - VaishnaviHire
-        - VannTen
-        - Xaenalt
-        - andrewballantyne
-        - atheo89
-        - dchourasia
-        - dfeddema
-        - dibryant
-        - durandom
-        - gmfrasca
-        - goern
-        - guimou
-        - harshad16
-        - heyselbi
-        - jedemo
-        - jeff-phillips-18
-        - jgarciao
-        - judyobrienie
-        - kywalker-rh
-        - lucferbux
-        - lugi0
-        - maroroman
-        - rimolive
-        - vaibhavjainwiz
-        privacy: closed
-        repos:
-          kserve: write
-      repo-modelmesh-write:
-        description: "Members with Write access to the modelmesh repository"
-        members:
-        - DaoDaoNoCode
-        - DharmitD
-        - HumairAK
-        - Jooho
-        - VaishnaviHire
-        - VannTen
-        - Xaenalt
-        - andrewballantyne
-        - atheo89
-        - dchourasia
-        - dfeddema
-        - dibryant
-        - durandom
-        - gmfrasca
-        - goern
-        - guimou
-        - harshad16
-        - heyselbi
-        - jedemo
-        - jeff-phillips-18
-        - jgarciao
-        - judyobrienie
-        - kywalker-rh
-        - lucferbux
-        - lugi0
-        - maroroman
-        - rimolive
-        - vaibhavjainwiz
-        privacy: closed
-        repos:
-          modelmesh: write
-      repo-modelmesh-runtime-adapter-write:
-        description: "Members with Write access to the modelmesh-runtime-adapter repository"
-        members:
-        - DaoDaoNoCode
-        - DharmitD
-        - HumairAK
-        - Jooho
-        - VaishnaviHire
-        - VannTen
-        - Xaenalt
-        - andrewballantyne
-        - atheo89
-        - dchourasia
-        - dfeddema
-        - dibryant
-        - durandom
-        - gmfrasca
-        - goern
-        - guimou
-        - harshad16
-        - heyselbi
-        - jedemo
-        - jeff-phillips-18
-        - jgarciao
-        - judyobrienie
-        - kywalker-rh
-        - lucferbux
-        - lugi0
-        - maroroman
-        - rimolive
-        - vaibhavjainwiz
-        privacy: closed
-        repos:
-          modelmesh-runtime-adapter: write
-      repo-modelmesh-serving-write:
-        description: "Members with Write access to the modelmesh-serving repository"
-        members:
-        - DaoDaoNoCode
-        - DharmitD
-        - HumairAK
+      Model Serving Maintainers:
+        description: "Maintainers for Model Serving related repos."
+        maintainers:
+        - taneem-ibrahim
```

Comment:
Can you order this in alphabetical order?

Reply:
done